### PR TITLE
Host unity packages on GitHub

### PR DESCRIPTION
**Pull request by aornelas**

### CODE REVIEW

**Comment by aornelas** [comment]
It's worth noting that

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+## Version 4.15.0 (June 29, 2017)
+- The MoPub Unity Plugin is now compatible with version 4.15 of the MoPub SDK.
+- The MoPub Unity Plugin is now available on GitHub.
+
+## Version 4.14
+- The MoPub Unity Plugin is now compatible with version 4.14 of the MoPub SDK.
+   
+## Version 4.13
+- The MoPub Unity Plugin is now compatible with version 4.13 of the MoPub SDK.
+
```
Comment (xsun) [inline CHANGELOG.md]: where's 4.12?

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+## Version 4.15.0 (June 29, 2017)
+- The MoPub Unity Plugin is now compatible with version 4.15 of the MoPub SDK.
+- The MoPub Unity Plugin is now available on GitHub.
+
+## Version 4.14
+- The MoPub Unity Plugin is now compatible with version 4.14 of the MoPub SDK.
```
Comment (xsun) [inline CHANGELOG.md]: are there no more details on these releases?

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+## Version 4.15.0 (June 29, 2017)
+- The MoPub Unity Plugin is now compatible with version 4.15 of the MoPub SDK.
```
Comment (xsun) [inline CHANGELOG.md]: add a .0 to all versions (SDK and unity plugin) for consistency

File: CHANGELOG.md (hunk 2 above)
Comment (aornelas) [inline CHANGELOG.md]: not AFAIK; I populated this from https://www.mopub.com/resources/docs/unity-engine-integration/

File: CHANGELOG.md (hunk 1 above)
Comment (aornelas) [inline CHANGELOG.md]: wasn't listed at https://www.mopub.com/resources/docs/unity-engine-integration/

File: CHANGELOG.md (hunk 3 above)
Comment (aornelas) [inline CHANGELOG.md]: ✅

**Comment by aornelas** [comment]
@xsun thanks! ready for another look

File: CHANGELOG.md (hunk 1 above)
Comment (xsun) [inline CHANGELOG.md]: we shipped 4.12.0. just add the entry and use the same description as others.

Comment (aornelas) [inline CHANGELOG.md]: ✅

**Comment by xsun** [comment]
lgtm

**Pull request merged**